plugins/catalog-backend-module-github/src/lib/github.ts

```diff
@@ -325,6 +334,66 @@ export async function getTeamMembers(
   return { members };
 }
 
+/**
+ *
+ * Gets the specified yaml files from a github repo and returns the yaml data so it can be used
+ *  in a provider to be turned into entities.
+ *
+ * @param client - An octokit graphql client
+ * @param org - The slug of the org to read
+ * @param files - List of file entries pulled from the config
+ */
+
+export async function getOrganizationEntities(
+  client: typeof graphql,
+  org: string,
+  files: Array<string>,
+): Promise<{ repositories: Array<Repository | RepositoryWithFiles> }> {
+  let queryFiles = ``;
+
+  for (const file of files) {
+    queryFiles += `
+    ${camelCase(file)}: object(expression: "HEAD:${file}") {
+      ... on Blob {
+        text
```

Comment:
We never end up using the text since this is just a presence check, thinking it would be better to instead of for the object ID or some other smaller piece of data.

Reply:
Good idea. Implemented.